app/src/main/java/com/cliqz/browser/widget/SearchBar.java

```diff
@@ -376,6 +376,8 @@ public void onFocusChange(View v, boolean hasFocus) {
 
             if (hasFocus) {
                 post(SearchBar.this::showKeyBoard);
+                bus.post(new Messages.DismissVpnPanel());
```

Comment:
Why two messages?

Reply:
We have to dismiss vpn panel or dashboard when we tap on url bar. Either of them could be open.